src/nestegg.c

```diff
@@ -312,6 +313,7 @@ struct nestegg {
 };
 
 struct nestegg_packet {
+  uint8_t keyframe_status;
```

Comment:
Put this at the end of the struct, otherwise it introduces 7 bytes of wasted padding.